frontend-web/webclient/app/Applications/Group.tsx

```diff
@@ -20,53 +25,67 @@ const ApplicationsGroup: React.FunctionComponent = () => {
         null
     );
 
+    const dispatch = useDispatch();
+    const [, invokeCommand] = useCloudCommand();
+
     useEffect(() => {
         fetchAppGroup(retrieveGroup({id: id}));
     }, [id]);
 
-    if (!appGroup.data) return <>Not found</>;
+    const favoriteStatus = useSelector<ReduxObject, ApplicationSummaryWithFavorite[]>(it => it.sidebar.favorites);
+
+    const onFavorite = useCallback(async (app: ApplicationSummaryWithFavorite) => {
+        const favoriteApp = favoriteStatus.find(it => it.metadata.name === app.metadata.name);
+        const isFavorite = favoriteApp !== undefined ? true : app.favorite;
+
+        dispatch(toggleAppFavorite(app, !isFavorite));
 
-    return <MainContainer 
-        header={
-            <>
-                <Flex justifyContent="space-between">
-                    <Heading.h2>
-                        <AppToolLogo name={appGroup.data.group.id.toString()} type="GROUP" size="45px" />
-                        {" "}
-                        {appGroup.data.group.title}
-                    </Heading.h2>
-                    <Box ml="auto" mt="30px">
-                        <ContextSwitcher />
-                    </Box>
-                </Flex>
-                <Absolute top="0px" left="calc(50% - 200px)">
-                    <LargeSearchBox />
-                </Absolute>
-            </>
+        try {
+            await invokeCommand(UCloud.compute.apps.toggleFavorite({
+                appName: app.metadata.name
+            }));
+        } catch (e) {
+            dispatch(toggleAppFavorite(app, !isFavorite));
         }
```

Comment:
I think running `invokeCommand` without passing a second argument, `invokeCommand` will do the error handling